.devcontainer/devcontainer.json

```diff
@@ -15,5 +15,10 @@
         "args": ["-l"]
       }
     }
-  }
+  },
+  "runArgs": [
+    // Allow node process to bind to port numbers below 1024 without root permissions
```

Comment:
```suggestion
```

Reply:
+1, I think JS comments are not valid JSON syntax.

Reply:
[VSCode uses jsonc for its config files](https://code.visualstudio.com/Docs/languages/json#_json-with-comments), so comments are not invalid in this case.